pkg/runtime/BUILD

```diff
@@ -9,8 +9,12 @@ load(
 
 go_library(
     name = "go_default_library",
-    srcs = ["doc.go"],
+    srcs = [
+        "doc.go",
+        "zz_generated.deepcopy.go",
```

Comment:
This doesn't look good.  What happened here?  @smarterclayton you had any trouble with code generating wrong? Is this the deepcopy thing you changed?